### DISCUSSION TITLE
Streams Debug [Feature Request]

### DISCUSSION THREAD

**Discussion by fravemel-upvnet** [microsoft/vscode-java-debug#1218] [Ideas]
First of all thanks, I use the debugger on daily bases, and I like it.
It would be nice to be able to debug java Streams (filter, map...) watch values on each iteration and the output of each function applied to the stream. I mean Trace values before and after each step in the Stream Chain.

**Comment by testforstephen** [comment]
@fravemel-upvnet This feature is interesting. I create an issue https://github.com/microsoft/vscode-java-debug/issues/1219 to track it. Pls vote there.
